Source/Fuse.Common/UpdateManager.uno

```diff
@@ -82,8 +81,18 @@ namespace Fuse
 		public List<UpdateListener> Onces = new List<UpdateListener>();
 		public List<UpdateListener> OncesPending = new List<UpdateListener>();
 		
-		public List<UpdateAction> PhaseDeferredActions = new List<UpdateAction>();
-		public int PhaseDeferredActionsAt = -1;
+		public Dictionary<int, Queue<UpdateAction>> PhaseDeferredActions = new Dictionary<int, Queue<UpdateAction>>();
+
+		public int HighestPriority
+		{
+			get
+			{
+				int max = -1;
+				foreach (var p in PhaseDeferredActions)
+					if (p.Value.Count > 0 && p.Key > max) max = p.Key;
```

Comment:
This looks wrong. The items with lower priority values are actually to be executed first (it's an odd history that resulted in this).